### PR TITLE
Random

### PR DESCRIPTION
**Pull request by mwarnck**

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -7,13 +7,14 @@ export default function Navigation() {
...
       <LinkButton to="/myDrinks">My Drinks</LinkButton>
+      <LinkButton to="/randomDrink">Random</LinkButton>
     </NavBar>
   );
 }
 
 const NavBar = styled.nav`
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (F-Kirchhoff) [inline src/components/Navigation.js:17]: you could use a repeat here
```suggestion
  grid-template-columns: repeat(4,1fr);
```

File: src/pages/RandomDrinkPage.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+const IngredientsList = styled.dl`
+  font-size: 0.9rem;
+`;
+
+const InstructionsContainer = styled.div`
+  display: grid;
+  gap: 10px;
+  margin: 5px 10px 20px;
+`;
```
Comment (F-Kirchhoff) [inline src/pages/RandomDrinkPage.js:220]: If you use these components in other pages as well, you could outsource them into another file and reuse them in all pages. By that, you dont have to rewrite the same component over and over if you want to change the styling. Or you try to generalize your drink page into one component.

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊 but maybe you can use your styled components more efficiently ;)

File: src/pages/RandomDrinkPage.js (hunk 2 above)
Comment (mwarnck) [reply src/pages/RandomDrinkPage.js:220]: Yes that is good point and I have planned a single user story for refactoring and styling the next days :)

**Pull request merged**